nsxt/resource_nsxt_policy_ipsec_vpn_service_test.go

```diff
@@ -0,0 +1,193 @@
+/* Copyright © 2022 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+var accTestPolicyIPSecVpnServiceCreateAttributes = map[string]string{
+	"display_name":  getAccTestResourceName(),
+	"description":   "terraform created",
+	"enabled":       "true",
+	"ha_sync":       "true",
+	"ike_log_level": "INFO",
+	"sources":       "192.168.10.0/24",
+	"destinations":  "192.169.10.0/24",
+	"action":        "BYPASS",
+}
+
+var accTestPolicyIPSecVpnServiceUpdateAttributes = map[string]string{
+	"display_name":  getAccTestResourceName(),
+	"description":   "terraform created",
+	"enabled":       "true",
+	"ha_sync":       "true",
+	"ike_log_level": "DEBUG",
+	"sources":       "192.170.10.0/24",
+	"destinations":  "192.171.10.0/24",
+	"action":        "BYPASS",
+}
+
+func TestAccResourceNsxtPolicyIPSecVpnService_basic(t *testing.T) {
+	testResourceName := "nsxt_policy_ipsec_vpn_service.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t); testAccOnlyLocalManager(t) },
+		Providers: testAccProviders,
+		CheckDestroy: func(state *terraform.State) error {
+			return testAccNsxtPolicyIPSecVpnServiceCheckDestroy(state, accTestPolicyIPSecVpnServiceUpdateAttributes["display_name"])
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: testAccNsxtPolicyIPSecVpnServiceMinimalistic(),
```

Comment:
It would be best to end the test with minimalistic config, since it validates unneeded attributes are deleted or reset back to default, which reveals bugs quite often.